model/columns_gen.go

```diff
@@ -110,12 +119,12 @@ func (c ColumnType) String() string {
 // If the type does not have a synonym then this method returns the receiver itself
 func (c ColumnType) SynonymType() ColumnType {
 	switch c {
+	case ColumnTypeInteger:
+		return ColumnTypeInt
 	case ColumnTypeNumeric:
 		return ColumnTypeDecimal
 	case ColumnTypeReal:
 		return ColumnTypeDouble
-	case ColumnTypeInteger:
-		return ColumnTypeInt
 	}
```

Comment:
maybe we should handle boolean type as tinyint(1) https://dev.mysql.com/doc/refman/5.6/ja/other-vendor-data-types.html